plumbing/object/change.go

```diff
@@ -78,6 +79,12 @@ func (c *Change) String() string {
 	return fmt.Sprintf("<Action: %s, Path: %s>", action, c.name())
 }
 
+// Patch returns a Patch with all the file changes in chunks. This
+// representation can be used to create several diff outputs.
+func (c *Change) Patch() (diff.Patcheable, error) {
```

Comment:
Will it make sense for Change to implement Patch or FilePatch instead of having a method that returns a Path?